packages/flutter/lib/src/gestures/multidrag.dart

```diff
@@ -26,6 +26,11 @@ export 'gesture_settings.dart' show DeviceGestureSettings;
 /// Signature for when [MultiDragGestureRecognizer] recognizes the start of a drag gesture.
 typedef GestureMultiDragStartCallback = Drag? Function(Offset position);
 
+/// Signature for [MultiDragGestureRecognizer._buttonsEventFilter].
```

Comment:
fly-by: Please don't reference private types in API docs.

Reply:
Thanks! Done.